cmd/boots/main.go

```diff
@@ -375,7 +378,7 @@ func registerInstallers() job.Installers {
 	o := osie.Installer{}
 	i.RegisterDistro("discovery", o.Discover())
 	// register osie as default
-	d := osie.Installer{}
+	d := osie.Installer{ExtraKernelArgs: strings.Split(cf.extraKernelArgs, ",")}
```

Comment:
it actually doesn't make any sense to split this at all, we don't ever interpret the args and don't care what format it is. I think doing the split will likely just do the wrong thing for args that themselves have a `,` (e.g.: `console=ttyS0,115200n8` or similar)

Reply:
ah, great point. will update.

Reply:
updated